torchtitan/config_manager.py

```diff
@@ -299,14 +299,14 @@ def __init__(self):
         self.parser.add_argument(
             "--experimental.pipeline_parallel_schedule",
             type=str,
-            choices=["1f1b", "gpipe", "interleaved_1f1b", "flexible_interleaved_1f1b"],
-            default="1f1b",
+            choices=["1F1B", "Gpipe", "Interleaved1F1B", "FlexibleInterleaved1F1B"],
```

Comment:
very tiny nit: `GPipe` instead of `Gpipe`

Reply:
updated